src/Authentication/OAuth10.php

```diff
@@ -49,16 +70,23 @@ public function __construct(array $authorization)
      */
     public function attachToClient(RestClient $client): void
     {
-        $sub = new Oauth1(
-            [
-            'consumer_key'    => $this->consumerKey,
+        $middleware = new Oauth1([
```

Comment:
Predtym aj teraz sa na to pouziva externa kniznica, ktora bola updatnuta na podporu noveho Guzzle.